frontend/src/components/activePage/TableInstance/TableInstanceLogic.tsx

```diff
@@ -104,8 +110,18 @@ const TableInstanceLogic: FC<ITableInstanceLogicProps> = ({ ...props }) => {
           return newItem;
         },
       });
+      return () => unsubscribe?.();
```

Comment:
Check if `unsubscribe` can actually be null, I suspect can't be...

Reply:
yes, it can't be, I've removed `.?`